README.md

```diff
@@ -103,6 +103,17 @@ To translate between LLVM IR and SPIR-V:
     * `-spirv-text` - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
     * `-help` - to see full list of options
 
+4. Multiple input files mode. Execute the following command to translate `input.bc` and `input1.bc` to `input.spv` and `input1.spv`.
+    ```
+    llvm-spirv input.bc input1.bc -o list.txt
```

Comment:
Probably it worth mentioning how this option affect other modes, i.e. `-r`, `-to-text`, `-to-binary`, etc.

Reply:
Mentioned.